broker/parameters.go

```diff
@@ -24,6 +26,7 @@ func parseUpdateParameters(data []byte) (*UpdateParameters, error) {
 	params := &UpdateParameters{}
 	err := unmarshalParameters(data, params, []string{
```

Comment:
I know you didn't write it but the existence of this `unmarshalParameters` function breaks my brain.

Reply:
+1